pkg/interactor/data.go

```diff
@@ -0,0 +1,59 @@
+package interactor
+
+import (
+	"strconv"
+	"time"
+
+	"github.com/CESARBR/knot-cloud-storage/pkg/data"
+	. "github.com/CESARBR/knot-cloud-storage/pkg/entities"
+)
+
+type DataInteractor struct {
+	DataStore data.DataStore
+}
+
+func NewDataInteractor(dataStore data.DataStore) DataInteractor {
+	return DataInteractor{dataStore}
+}
+
+func (d *DataInteractor) GetAll(order int, skip int, take int, startDate time.Time, finishDate time.Time) ([]Data, error) {
+
+	selectOrder := "timestamp"
+	if order == -1 {
+		selectOrder = "-timestamp"
+	}
+
+	data, err := d.DataStore.Get(selectOrder, skip, take, startDate, finishDate)
+
+	return data, err
```

Comment:
Error handlinggg :stuck_out_tongue: 

Also, even though I might not have commented this before in other places this might've occurred, when a method or function returns and `error`, you should always handle then `return err` or `return "", err`. The **last** return statement should `return nil` or `return something, nil` :woman_astronaut:  

```suggestion
	data, err := d.DataStore.Get(selectOrder, skip, take, startDate, finishDate)
  if err != nil {
    // do something
    return "", err
  }
  
	return data, nil
```